src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -17,10 +17,12 @@ use kbs_protocol::message::*;
 
 pub struct Kbc {
     tee: String,
+    kbs_uri: String,
     token: Option<String>,
     nonce: String,
     tee_key: Option<TeeKey>,
     attester: Option<Box<dyn Attester + Send + Sync>>,
+    kbs_client: Option<reqwest::Client>,
```

Comment:
why is this an Option?
can this Kbc do anything without a properly initialized `kbs_client` ?